atlasdb-cli/src/test/java/com/palantir/atlasdb/cli/command/TestSingleBackendCommand.java

```diff
@@ -71,9 +72,9 @@ public int execute(AtlasDbServices services) {
                 services.getKeyValueService().createTable(table, AtlasDbConstants.GENERIC_TABLE_METADATA);
                 Preconditions.checkArgument(
                         services.getKeyValueService().getAllTableNames().contains(table),
-                        "kvs contains tables %s, but not table %s",
-                        services.getKeyValueService().getAllTableNames(),
-                        table.getQualifiedName());
+                        "kvs contains tables, but not table",
+                        UnsafeArg.of("tables", services.getKeyValueService().getAllTableNames()),
+                        UnsafeArg.of("table", table.getQualifiedName()));
```

Comment:
Probably worth using `LoggingArgs` for table-related things: there _may_ be some metadata we can use about which tables are safe or unsafe (e.g. all atlas defined internal tables are safe). In particular `LoggingArgs.tableRef(...)` or 
 `SafeArg.of(name, LoggingArgs.safeTablesOrPlaceholder(...)` might be useful